mteb/tasks/Retrieval/eng/TopiOCQARetrieval.py

```diff
@@ -0,0 +1,102 @@
+from datasets import load_dataset
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+CORPUS_HF_NAME, CORPUS_HF_VERSION, CORPUS_HF_SPLIT = (
+    "McGill-NLP/TopiOCQA-wiki-corpus",
+    "50ae3b82713b1a935190def03ce7e7e75a318636",
+    "train",
+)
+
+
+class TopiOCQARetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="TopiOCQA",
+        dataset={
+            "path": "McGill-NLP/TopiOCQA",
+            "revision": "66cd1dbf5577c653ecb99b385200f08e15e12f30",
+        },
+        reference="https://mcgill-nlp.github.io/topiocqa",
+        description=(
+            "TopiOCQA (Human-in-the-loop Attributable Generative Retrieval for Information-seeking Dataset)"
+            "is information-seeking conversational dataset with challenging topic switching phenomena."
+            "It consists of conversation histories along with manually labelled relevant/gold passage."
+        ),
+        type="Retrieval",
+        category="s2p",
```

Comment:
This is not exactly accurate but currently there is no other better suited option